website_oca_integrator/views/website_oca_integrator_templates.xml

```diff
@@ -0,0 +1,132 @@
+
+                <!--Display integrators by contry wise -->
+                <div class="col-md-3 mb32" id="integrator_left">
+                    <ul id="integrators_countries" class="nav nav-pills nav-stacked mt16">
+                        <li class="nav-header"><h3>Filter by Country</h3></li>
+                        <t t-foreach="countries" t-as="country">
+                            <li t-if="country['country_id']" t-att-class="country['active'] and 'active' or None">
+                                <a t-attf-href="/integrators#{country['country_id'][0] and '/country/%s' % country['country_id'][0] or '' }#{ '?' + (search_path or '') + (country['country_id'][0] == 0 and '&amp;country_all=True' or '')}">
+                                    <span class="badge pull-right" t-esc="country['country_id_count'] or ''"/>
+                                    <t t-esc="country['country_id'][1]"/>
+                                </a>
+                            </li>
+                        </t>
+                    </ul>
+                </div>
+
+                <!--Integrators content -->
+                <div class="col-md-8 col-md-offset-1" id="ref_content">
+                    <div class='navbar'>
+                        <div>
+                            <t t-call="website.pager"/>
+                            <form action="" method="get" class="navbar-search pull-right pagination form-inline">
+                                <div class="form-group pull-right">
+                                    <input type="text" name="search" class="search-query col-md-2 mt4 form-control" placeholder="Search" t-att-value="searches.get('search', '')"/>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                    <div>
+                        <p t-if="not integrators">No result found</p>
+                        <t t-foreach="integrators" t-as="integrator">
+                            <div class="media">
+                                <a class="pull-left" t-attf-href="/integrators/#{slug(integrator)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}"
+                                   t-field="integrator.image"
+                                   t-options='{"widget": "image", "class": "media-object", "max_width": 128}'
+                                />
+
+                                <div class="media-body o_partner_body" style="min-height: 64px;">
+                                    <a class="media-heading" t-attf-href="/integrators/#{slug(integrator)}?#{current_grade and 'grade_id=%s&amp;' % current_grade.id}#{current_country and 'country_id=%s' % current_country.id}">
+                                        <span t-field="integrator.display_name"/>
+                                    </a>
+                                    <t t-if="integrator.grade_id.name">
```

Comment:
This is incorrect for OCA, as integrator.grade_id is a sponsorship level.  Hard to see without demo system what was meant here, but maybe the integrator span should be sponsor.